src/tests/issues.targets

```diff
@@ -66,6 +66,9 @@
         <ExcludeList Include="$(XunitTestBinBase)/GC/Regressions/Github/Runtime_76219/Runtime_76219/*">
             <Issue>https://github.com/dotnet/runtime/issues/78899</Issue>
         </ExcludeList>
+        <ExcludeList Include="$(XunitTestBinBase)/JIT/Regression/JitBlue/Runtime_82535/*">
+          <Issue>https://github.com/dotnet/runtime/pull/82663</Issue>
```

Comment:
Is there a tracking issue for the CoreCLR problem with this test? We don't typically put pull requests into the Issue field because Mono pull requests are not CoreCLR tracking issues that someone would look at.

Cc @mangod9 @janvorli @dotnet/jit-contrib

Reply:
Will make one.